test/workflows/components/workflows.libsonnet

```diff
@@ -240,6 +257,21 @@
             $.parts(namespace, name, overrides).e2e(prow_env, bucket).buildTemplate("unit-test", testWorkerImage, [
               "test/scripts/unit-test.sh",
             ]),  // unit test
+            $.parts(namespace, name, overrides).e2e(prow_env, bucket).buildTemplate("pylint-checking", testWorkerImage, [
+              "python",
+              "-m",
+              "kubeflow.testing.test_py_lint",
+              "--artifacts_dir=" + artifactsDir,
+              "--src_dir=" + pylintSrcDir,
+            ]),  // pylint-checking
+            $.parts(namespace, name, overrides).e2e(prow_env, bucket).buildTemplate("copy-artifacts", testWorkerImage, [
```

Comment:
The `copy-artifacts` will copy and check with result. Otherwise, from kubeflow `kubeflow.testing.test_py_lint` in `pylint-checking` will pass even if there are some lint errors. From `kubeflow.testing.test_py_lint` implements, that only run and write logs, and exist with 0 even if some lint errors.